spec/migrations/20150817213409_clear_tenant_seed_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "spec_helper"
+require Rails.root.join("db/migrate/20150817213409_clear_tenant_seed")
+
+describe ClearTenantSeed do
+  class ClearTenantSeed::Tenant < ActiveRecord::Base
+  end
+  let(:tenant_stub) { migration_stub(:Tenant) }
```

Comment:
I really don't like that the migration calls raw SQL and not using AR models.  If you use AR models, you should have a stub.